ref/appliance-command-reference/gateway.md

```diff
@@ -0,0 +1,34 @@
+The `gateway` command allows and manages access to the HTTP API of the stream gateway. If the HTTP API is exposed (which is the default), authenticated clients can connect to it.
+
+    anbox-cloud-appliance gateway <subcommand>
+
+## Subcommands
+
+### `account`
+Manages accounts to access the Anbox stream gateway.
+
+    anbox-cloud-appliance gateway account <subcommand>
+
+#### Subcommands
+
+##### `create`
+Creates an account to access the Anbox stream gateway.
+
+    anbox-cloud-appliance gateway account create <account-name>
+
+This command creates an account with the given <account-name> and returns a token which you can use for accessing the Anbox Stream Gateway HTTP API directly or through the Anbox Stream SDK. See [How to access the stream gateway](https://discourse.ubuntu.com/t/managing-stream-gateway-access/17784) for more information.
+
+##### `delete`
+Deletes the specified account and removes access to the Anbox stream gateway for the specified account.
+
+    anbox-cloud-appliance gateway account delete <account-name>
+
```

Comment:
Do you think it would make sense to combine the subcommands?
Instead of having the `account` command, with two subcommands, can't we show that as two different commands: `account create` and `account delete`? It's not technically how it's implemented.

```suggestion
### `account create`
Creates an account to access the Anbox stream gateway.

    anbox-cloud-appliance gateway account create <account-name>

This command creates an account with the given <account-name> and returns a token which you can use for accessing the Anbox Stream Gateway HTTP API directly or through the Anbox Stream SDK. See [How to access the stream gateway](https://discourse.ubuntu.com/t/managing-stream-gateway-access/17784) for more information.

### `account delete`
Deletes the specified account and removes access to the Anbox stream gateway for the specified account.

    anbox-cloud-appliance gateway account delete <account-name>

```

This removes two heading levels and keep the important information. What do you think?

Reply:
How does it read now? I kind of did what you have suggested but not entirely. I think it would be useful to retain the idea that the `create` and `delete` are subcommands to help the readers understand. At the same time, I do see your point about too many levels. Let me know what you think about the latest update and if it's good, I will implement the same everywhere there is a third level of subcommands.

Reply:
This is great, thanks!